test/python/opflow/test_gradients.py

```diff
@@ -685,6 +686,87 @@ def test_natural_gradient4(self, grad_method, qfi_method, regularization):
         except MissingOptionalLibraryError as ex:
             self.skipTest(str(ex))
 
```

Comment:
Could you add some tests for
* setting the `phase_fix` argument
* setting the `aux_meas_op` to something unsupported (like `Y`, `1j * Y` or just `X`)
* triggering the errors/warnings in the natural gradient combo fn

? 🙂